docs/schema.json

```diff
@@ -145,7 +145,7 @@
               "default": false
             },
             "separate_signature": {
-              "title": "Whether to put the whole signature in a code block below the heading. If Black is installed, the signature is also formatted using it.",
+              "title": "Whether to put the whole signature in a code block below the heading. If a formatter (Black or ruff) is installed, the signature is also formatted using it.",
```

Comment:
Seems to be written Ruff, with a capital R, could you update all occurrences?

Reply:
Great catch - I've updated the commit